wiring/src/spark_wiring_string.cpp

```diff
@@ -72,6 +72,12 @@ String::String(const char *cstr)
 	if (cstr) copy(cstr, strlen(cstr));
 }
 
+String::String(const char *cstr, unsigned int length)
```

Comment:
Please add a `wiring/no_fixture/string.cpp` test for String::String(const char *cstr, unsigned int length)